jujugui/static/gui/src/app/app.js

```diff
@@ -599,6 +599,8 @@ YUI.add('juju-gui', function(Y) {
         if (!evt.newVal) {
           // The model is not connected, do nothing waiting for a reconnection.
           console.log('model disconnected');
+          // Need to re-render the provider logo so that it clears.
+          this._renderProviderLogo();
```

Comment:
It seems that calling from `_renderComponents` is necessary to bootstrap it. Otherwise the logo never shows.